lib/rss/itunes.rb

```diff
@@ -357,10 +357,10 @@ def construct(hours, minutes, seconds)
           if components.include?(nil)
             nil
           else
-            components.unshift(hours) if hours and hours > 0
+            components.unshift(hours) if hours&.positive?
```

Comment:
```suggestion
            components.unshift(hours) if hours and hours > 0
```

Not in `Style/StringLiterals`.